src/openshift/component.ts

```diff
@@ -28,16 +28,32 @@ export class Component extends OpenShiftItem {
         }
     }
 
+    static async delComponent () {
```

Comment:
This method should not be in the API. Either move the code into del, or make it private and then call it from del.

Reply:
I did not realize it is possible to register the same method twice with different names:
* `OpenShift: Delete Component` for palette
* `Delete` for `OpenShift Application Explorer`

Example is here https://github.com/redhat-developer/vscode-openshift-tools/pull/297/files#diff-85d649dcccfbc33846ac83964c2de415R32.